packages/vc-verification/package.json

```diff
@@ -27,6 +36,33 @@
   },
   "license": "GPL-3.0-or-later",
   "dependencies": {
-    "ethers": "^5.5.4"
+    "@energyweb/onchain-claims": "1.0.0",
+    "@energyweb/credential-governance": "1.0.0",
+    "ethers": "^5.6.1"
+  },
+  "devDependencies": {
+    "@ew-did-registry/claims": "0.6.3-alpha.527.0",
+    "@ew-did-registry/credentials-interface": "0.6.3-alpha.527.0",
+    "@ew-did-registry/did-ethr-resolver": "0.6.3-alpha.527.0",
+    "@ew-did-registry/did-ipfs-store": "0.6.3-alpha.527.0",
+    "@ew-did-registry/did-registry": "0.6.3-alpha.527.0",
+    "@ew-did-registry/did-store-interface": "0.6.3-alpha.527.0",
+    "@typechain/ethers-v5": "^9.0.0",
+    "chai": "4.3.4",
+    "chai-arrays": "2.2.0",
+    "chai-as-promised": "7.1.1",
+    "eth-ens-namehash": "^2.0.8",
+    "ganache-cli": "^6.12.2",
+    "ipfs": "^0.58.3",
+    "ipfsd-ctl": "3.0.0",
+    "ipfs-http-client": "^43.0.0",
+    "js-sha3": "^0.8.0",
+    "jsonwebtoken": "^8.5.1",
+    "mocha": "7.2.0",
+    "run-with-testrpc": "0.3.1",
+    "truffle": "5.4.29",
+    "ts-node": "9.1.1",
+    "typechain": "7.0.0",
+    "typescript": "^4.5.5"
```

Comment:
We missing https://github.com/energywebfoundation/shared-configs

Reply:
It is at the repo level https://github.com/energywebfoundation/ew-credentials/blob/17d84c97560bca4028a9341bf60fec5c34fc1e49/package.json#L48, though the version needs to be updated.

Reply:
fair enough